### PR TITLE
Manga chapters

### PR DESCRIPTION
**Pull request by matthewdias**

### CODE REVIEW

File: app/models/chapter.rb [hunk 1]
```diff
@@ -22,8 +26,20 @@
...
 class Chapter < ApplicationRecord
+  include Titleable
+
   belongs_to :manga
 
+  has_attached_file :thumbnail
+
   validates :manga, presence: true
   validates :number, presence: true
+  # validates :volume_number, presence: true
```
Comment (matthewdias) [inline app/models/chapter.rb]: do we want to validate this? i'm thinking sometimes we might want to add chapters when volumes aren't out yet.

Comment (toyhammered) [reply app/models/chapter.rb]: nope don't validate

Comment (NuckChorris) [reply app/models/chapter.rb]: (probably wanna remove this line entirely)

**Review by NuckChorris — COMMENTED** [review]
Looks awesome, let me know when this is ready for merge!

**Pull request merged**